.github/workflows/code-quality.yml

```diff
@@ -1,15 +1,15 @@
 # yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
-name: Offchain code quality assessment
+name: Rust code quality assessment
```

Comment:
```suggestion
name: Assess Rust code quality
```

Reply:
It also makes sense to rename the file to `rust-code-quality.yml`, don't you think?